src/native-shim.js

```diff
@@ -149,7 +149,16 @@
     constructorByTagname.set(tagname, elementClass);
     nativeDefine.call(window.customElements, tagname, StandInElement);
   };
-
-  window.customElements.get = (tagname) => constructorByTagname.get(tagname);
-
+  
+  let get = (tagname) => constructorByTagname.get(tagname);
+  
+  // Workaround for Safari bug where patching customElements can be lost, likely
+  // due to native wrapper garbage collection issue
+  Object.defineProperty(window, 'customElements',
```

Comment:
This is the strangest of the three lines - it should be a no-op. Is it required for the workaround?